src/components/dumb/OeSelect.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div v-click-outside="hideResults" @click="showResults = !showResults" class="js-vl-select" role="listbox"
+    data-type="select-one" tabindex="0" dir="ltr">
+    <div class="vl-select__inner">
+      <div class="vl-input-field">
+        <div class="vl-select__item vl-select__item--selectable" :data-id="selectOptionLabel"
+          :data-value="selectedOption">
+          {{ selectOptionLabel }}
+        </div>
+      </div>
+    </div>
+    <div v-if="showResults" class="vl-select__list vl-select__list--dropdown">
+      <div class="vl-select__list" dir="ltr" role="listbox">
+        <template v-for="option in options">
```

Comment:
Bij een `v-for` wordt een `:key` verwacht zodat elk element als uniek gezien kan worden